scripts/microbe_stage/microbe.lua

```diff
@@ -690,32 +743,39 @@ function Microbe:kill()
     self.microbe.movementDirection = Vector3(0,0,0)
     self.rigidBody:clearForces()
     microbeSceneNode.visible = false
+    --[[ since other microbes can kll each other now, this is deprecated
```

Comment:
@jjonj what do you think we should do with the victory message? I say it has outlived its usefulness